adapters/repos/db/lsmkv/segment_group.go

```diff
@@ -710,6 +711,18 @@ func (sg *SegmentGroup) count() int {
 	return count
 }
 
+func (sg *SegmentGroup) payloadSize() int64 {
+	segments, release := sg.getAndLockSegments()
+	defer release()
+
+	totalSize := int64(0)
+	for _, seg := range segments {
+		totalSize += int64(seg.PayloadSize())
```

Comment:
segments consists of header, payload and binary index (used to navigate quickly over the payload). What `PayloadSize()` method is expected to return? Based on the following test (`dataEndPos`) it returns header size + payload size.

Reply:
we want to measure the disk space used by the data portion of the segment. maybe `payloadSize` could be misleading we could call `ContentSize()` wdyt ?

Reply:
we want the whole storage in disk despite the content

Reply:
it does call the segments method [under the hood](https://github.com/weaviate/weaviate/blob/moogacs-4819c563/adapters/repos/db/lsmkv/segment_group.go#L475)